src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,147 @@
+| [test](https://pub.dartlang.org/packages/test) | A standard way of writing and running tests in Dart. | expect(), group(), test() |
+{:.table .table-striped .nowrap}
+
+To find more packages, see [pub.dartlang.org](https://pub.dartlang.org/).
+
+## Packages that correspond to SDK libraries
+
+These "expansion pack" libraries have the same name as their counterpart in the SDK.
+Each of these libraries builds upon an SDK library, adding
+additional functionality and filling in missing features:
+
+| **Package** | **Description** | **Commonly used APIs** |
+| [async](https://www.dartdocs.org/documentation/async/latest/) | Utility classes to work with asynchronous computations. For more information, see [Unboxing Packages: async part 1](http://news.dartlang.org/2016/03/unboxing-packages-async-part-1.html), [part 2](http://news.dartlang.org/2016/03/unboxing-packages-async-part-2.html), and [part 3.](http://news.dartlang.org/2016/04/unboxing-packages-async-part-3.html) | AsyncMemoizer, CancelableOperation, FutureGroup, LazyStream, Result, StreamCompleter, StreamGroup, StreamSplitter |
+| [collection](https://www.dartdocs.org/documentation/collection/latest) | Utility functions and classes to make working with collections easier. For more information, see [Unboxing Packages: collection.](http://news.dartlang.org/2016/01/unboxing-packages-collection.html) | Equality, CanonicalizedMap, MapKeySet, MapValueSet, PriorityQueue, QueueList |
+|[convert](https://www.dartdocs.org/documentation/convert/latest/) | Contains encoders and decoders for converting between different data representations. Note that one of the data representations is commonly known as URL-encoded (named `percent` here). | HexDecoder, PercentDecoder |
+{:.table .table-striped .nowrap}
+
+## Specialized libraries
+
+This page doesn't include some of the more specialized libraries that are covered
+elsewhere.
+
+### Web libraries
+
+If you write web apps, check out Angular 2, a web application framework.
+Other available resources are the
+[js](https://www.dartdocs.org/documentation/js/latest) package
+for interoperability with JavaScript APIs, and the
+[dart:html](https://api.dartlang.org/stable/dart-html/dart-html-library.html)
+library for low-level HTML programming.<br>
+More information: [webdev.dartlang.org]({{site.webdev}})
+
+### Flutter libraries
+
+Use Flutter to write mobile applications.
+The core libraries distributed with the Flutter SDK are documented at
+[docs.flutter.io](http://docs.flutter.io/). To import these libraries,
+follow the instructions in [Importing libraries from
+packages](https://www.dartlang.org/tools/pub/get-started#importing-libraries-from-packages).
+Additional third-party libraries for Flutter can be found on GitHub.
+For information about how to use a shared library,
+see its README documentation on GitHub.<br>
```

Comment:
I'm not crazy about this, either. (It seems too specific for this page. Also, the term "shared library" seems ambiguous—a library shared with the community, or a shared vs. static library?) Can you instead link to a place on flutter.io that discusses these libraries?